TypeScript/src/components/Part-A.ts

```diff
@@ -0,0 +1,121 @@
+import { Order } from './Order';
+import sortBy from 'lodash.sortby';
+import { function1PartA } from '../output/Output';
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+type MaybeNumber = number | null;
+
+const staticOrders = () : Order[] =>{
```

Comment:
can you explain in english what this means?

Reply:
I called it static Orders because we have an specific array [52,208...] that we are manipulating it and not taking a new Orders like in Part B